build/dependencies.props

```diff
@@ -1,12 +1,13 @@
 <Project>
-  <PropertyGroup Label="SDK Versions">    
+  <PropertyGroup Label="SDK Versions">
     <NetFrameworkVersion>net5.0</NetFrameworkVersion>
     <NetStandardVersion>netstandard2.0</NetStandardVersion>
     <MicrosoftNETTestSdk>16.8.0</MicrosoftNETTestSdk>
     <LangVersion>latest</LangVersion>
   </PropertyGroup>
 
   <PropertyGroup Label="Package information">
+    <PackageId>AspNetCore.$(MSBuildProjectName)</PackageId>
```

Comment:
@unaizorrilla This will not copy the configuration among dozens of projects. Alas, for two of them the authors did not comply with the naming conventions and gave different names to their packages.